ansible_base/jwt_consumer/common/auth.py

```diff
@@ -173,26 +180,31 @@ def map_user_fields(self):
                 logger.info(f"Saving user {self.user.username}")
                 self.user.save()
 
-    def validate_token(self, unencrypted_token, decryption_key):
+    def validate_token(self, unencrypted_token, decryption_key, request_id=None):
         validated_body = None
 
         local_required_field = ["sub", "user_data", "exp", "objects", "object_roles", "global_roles", "version"]
 
         # Decrypt the token
         try:
             logger.info("Decrypting token")
-            validated_body = jwt.decode(
+            validated_body = self.decode_jwt_token(
                 unencrypted_token,
                 decryption_key,
-                audience="ansible-services",
                 options={"require": local_required_field},
```

Comment:
Should we move this to be the default in `decode_jwt_token`? That method could take an argument like "additional options" and do an `options.update(additional_options)`.

Reply:
Refactored.